lib/matplotlib/quiver.py

```diff
@@ -85,6 +85,13 @@
     height of the axes, respectively, when the the window is resized;
     for 'dots' or 'inches', resizing does not change the arrows.
 
+  *orientation*: ['absolute' | 'relative']
```

Comment:
Good question.  The only case I came up with is if I am plotting velocity profiles versus time, for example, with depth positive down. Then I would want my U and V (or U and W if I actually had usable vertical velocity estimates) to remain in normal positive-right, positive-up axes.  This is an unusual case.  It really seems the default should *not* be the present behavior, and it's not even clear to me now that the kwarg is needed.  This is one of those potentially awkward transitions.  How many users are relying on the present behavior?  Probably not many, but there may be some who are using workarounds, which would break if the default behavior were changed.